romeo/src/state.rs

```diff
@@ -37,18 +42,31 @@ pub struct DepositInfo {
     pub amount: u64,
 
     /// Recipient of the sBTC
-    pub recipient: StacksAddress,
-
-    /// Name of the contract where the funds should be minted
-    pub contract_name: ContractName,
+    pub recipient: PrincipalData,
 
-    /// Height of the Bitcoin blockchain where this tx is included
-    pub block_height: u64,
+    /// Height of the Bitcoin blockchain where this deposit transaction exists
+    pub block_height: u32,
 }
 
 impl Deposit {
-    fn mint(&mut self, _block_height: u64) -> Option<Task> {
-        todo!();
+    fn mint(&self) -> Option<Task> {
+        match self.mint.as_ref() {
+            Some(Response {
+                status: TransactionStatus::Broadcasted,
+                txid,
+            }) => Some(Task::CheckStacksTransactionStatus(*txid)),
+            // TODO: Think about removing deposits at this stage
```

Comment:
IMO TODOs should always have a corresponding ticket so that if we decide never to do it or do decide to do it the effort is tracked. What are your thoughts?

Reply:
Agreed. I've tried to enforce it throughout the code base but occasionally something slips through.

Reply:
Created a separate issue https://github.com/stacks-network/sbtc/issues/99.

Reply:
And added the issue number to the TODO comment.